### DISCUSSION TITLE
My page can't show the theme

### DISCUSSION THREAD

**Discussion by 0akalin0** [community/community#22379] [Other Feature Feedback, Questions, & Ideas]
<p>hi all, this is my page at <a href="https://0akalin0.github.io/" rel="noopener nofollow ugc">https://0akalin0.github.io/</a><br>
Now the page can’t show the theme, but it can work locally just like below.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/b/8b6920bf8954bc71926a9415c4380e04f84132e8.png" data-download-href="/uploads/short-url/jThBruUKjB5B553jX1wVVn6zj1S.png?dl=1" title="2021-12-08 (2)" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/95558172/181070657-926df758-3d20-49e1-8641-e2760d81ffc9.png" alt="2021-12-08 (2)" data-base62-sha1="jThBruUKjB5B553jX1wVVn6zj1S" width="345" height="174" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/b/8b6920bf8954bc71926a9415c4380e04f84132e8_2_345x174.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/b/8b6920bf8954bc71926a9415c4380e04f84132e8_2_517x261.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/8/b/8b6920bf8954bc71926a9415c4380e04f84132e8_2_690x348.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/b/8b6920bf8954bc71926a9415c4380e04f84132e8_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">2021-12-08 (2)</span><span class="informations">1803×912 51.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div><br>
I’ve read through all the similar problem’s solutions, and I’ve tried to change the url and root in _config.yml, but it still couldn’t work.<br>
Here is what it looks like now.
<pre><code class="lang-auto"># URL
## Set your site url here. For example, if you use GitHub Page, set url as 'https://username.github.io/project'
url: https://0akalin0.github.io/0akalin0.github.io
root: /0akalin0.github.io/
</code></pre>
<p>The repo is here:<br>
<a href="https://github.com/0akalin0/0akalin0.github.io" rel="noopener nofollow ugc">https://github.com/0akalin0/0akalin0.github.io</a></p>
<p>Thank you!</p>

**Comment by yoannchaudet** [comment]
<p>Hello and welcome here!</p>
<p>Have you tried this instead?</p>
<pre><code class="lang-auto"># URL
url: https://0akalin0.github.io
root: /
</code></pre>

**Comment by 0akalin0** [comment]
<p>It can work now!<br>
Thanks a lot for your help!</p>
